### PR TITLE
Add digital-credential-get experimental permission to features.md

### PR DESCRIPTION
**Pull request by pkotwicz**
Github issue for context https://github.com/WICG/digital-credentials/issues/78

### CODE REVIEW

**Comment by clelland** [comment]
I don't see a feature for this in the explainer, or in Chromium source code anywhere -- is this implemented or proposed yet?

File: features.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ experimentation by web developers.
 | ------------ | ------- | --------------- |
 | `browsing-topics` | [Explainer](https://github.com/jkarlin/topics/) | Status "[Started](https://bugs.chromium.org/p/chromium/issues/detail?id=1294456)" in Chrome |
 | `conversion-measurement ` | [Explainer](https://github.com/WICG/conversion-measurement-api#publisher-controls-for-impression-declaration) | Experimental in Chrome<sup>[3](#fn3)</sup> |
+| `digital-credential-get` | [Explainer](https://wicg.github.io/digital-credentials/) | Behind a flag in Chrome<sup>[6](#fn6)</sup>
```
Comment (samuelgoto) [inline features.md:83]:
```suggestion
| `digital-credentials-get` | [Explainer](https://wicg.github.io/digital-credentials/) | Behind a flag in Chrome<sup>[6](#fn6)</sup>
```

For consistency with WebAuthn and FedCM, I think we want the plural:

https://github.com/w3c/webappsec-permissions-policy/blob/main/features.md

**Comment by pkotwicz** [comment]
> I don't see a feature for this in the explainer, or in Chromium source code anywhere -- is this implemented or proposed yet?

Yes. This is implemented behind the WebIdentityDigitalCredentials feature
Code for permissions policy https://chromium-review.googlesource.com/c/chromium/src/+/5678955

**Comment by clelland** [comment]
Thanks! I'm happy to merge this as soon as that CL lands then.

**Pull request merged**